scripts/set_xtensa_params.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright(c) 2023 Intel Corporation. All rights reserved.
+
+set -e
+
+platform=$1
+
+case ${platform,,} in
```

Comment:
??

```suggestion
case ${platform} in
```

Reply:
OK I found that this does: lowercasing. I don't see why we need that here, we're case sensitive everywhere else.

It's a relatively new and pretty cryptic bash feature, need at the very least a comment explaining what it is and why it's needed (if actually needed)